### ISSUE TITLE
java.lang.NoClassDefFoundError: com/jcraft/jsch/JSchException

### ISSUE THREAD

**Issue by FabHan** [bazelbuild/bazel#586]
I tried to build `TensorFlow` from source, following [this instruction](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md). However, I failed at this step.
```
➜  tensorflow git:(master) bazel build -c opt //tensorflow/cc:tutorials_example_trainer
..........
INFO: Elapsed time: 2.016s
java.lang.NoClassDefFoundError: com/jcraft/jsch/JSchException
        at org.eclipse.jgit.transport.JschConfigSessionFactory.getSession(JschConfigSessionFactory.java:109)
        at org.eclipse.jgit.transport.SshTransport.getSession(SshTransport.java:136)
        at org.eclipse.jgit.transport.TransportGitSsh$SshFetchConnection.<init>(TransportGitSsh.java:262)
        at org.eclipse.jgit.transport.TransportGitSsh.openFetch(TransportGitSsh.java:161)
        at org.eclipse.jgit.transport.FetchProcess.executeImp(FetchProcess.java:136)
        at org.eclipse.jgit.transport.FetchProcess.execute(FetchProcess.java:122)
        at org.eclipse.jgit.transport.Transport.fetch(Transport.java:1138)
        at org.eclipse.jgit.api.FetchCommand.call(FetchCommand.java:130)
        at org.eclipse.jgit.api.CloneCommand.fetch(CloneCommand.java:193)
        at org.eclipse.jgit.api.CloneCommand.call(CloneCommand.java:133)
        at com.google.devtools.build.lib.bazel.repository.GitCloneFunction.compute(GitCloneFunction.java:124)
        at com.google.devtools.build.skyframe.ParallelEvaluator$Evaluate.run(ParallelEvaluator.java:725)
        at com.google.devtools.build.lib.concurrent.AbstractQueueVisitor$2.run(AbstractQueueVisitor.java:436)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: com.jcraft.jsch.JSchException
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 16 more
```

I posted [this issue on TensorFlow](https://github.com/tensorflow/tensorflow/issues/105) but the TensorFlow team thinks that it is a bazel problem so I'm reporting the issue here.

**Comment by davidzchen** [comment]
Hi @FabHan - Thanks for reporting this. I will take a look. Which OS are you observing this on?

**Comment by FabHan** [comment]
I'm using Ubuntu Trusty. The bazel version is 0.1.0 (per `TensorFlow` instructions).

**Comment by FabHan** [comment]
FYI, without `git checkout tags/0.1.0`, I got pass this state, but ran into another problem (reported [here](https://github.com/tensorflow/tensorflow/issues/119)) when building protobuf.

**Comment by davidzchen** [comment]
Hi @FabHan, the commit that fixed this is 916370ec6195d7fc13812e64a65c2cd6ef9e36c9, which is not in the 0.1.0 release. The 0.1.0 release is actually rather old (cut back in September).

I was able to build TensorFlow using the 0.1.1 Bazel release. You can try `git checkout tags/0.1.1` as a workaround.

@damienmg, @kchodorow - Perhaps we should have the TensorFlow docs updated to use a newer release of Bazel?

**Comment by damienmg** [comment]
Marking this issue as resolved as it is a problem with tensorflow documentation.
